ush/generate_FV3LAM_wflow.sh

```diff
@@ -228,6 +228,7 @@ settings="\
   'wtime_make_lbcs': ${WTIME_MAKE_LBCS}
   'wtime_run_fcst': ${WTIME_RUN_FCST}
   'wtime_run_post': ${WTIME_RUN_POST}
+  'wtime_upload': ${WTIME_RUN_POST}
```

Comment:
Should this use a new config option `WTIME_UPLOAD`?

Reply:
That's an excellent question! I should definitely go ahead and fix that the right way.